impeller/entity/contents/contents.cc

```diff
@@ -86,10 +92,17 @@ std::optional<Snapshot> Contents::RenderToSnapshot(
       [&contents = *this, &entity, &coverage](const ContentContext& renderer,
                                               RenderPass& pass) -> bool {
         Entity sub_entity;
+        auto color_size = contents.GetColorSourceSize();
```

Comment:
I don't think this is quite right. Why would we need to perform a scale-up on the entity before rendering it here?

Reply:
Yes this is the problem.

Reply:
The idea was this would provide corrective scaling if the color source was down-scaled for coverage hinting we'd scale it to the right coordinate space again.

Reply:
This is gone now. Instead I'm overriding TiledTextureContents::RenderToSnapshot to pass through the original texture in cases where theeffect matrix is identity.